docker-compose-with-jupyter.yml

```diff
@@ -2,13 +2,12 @@ version: "3"
 
 services:
   jupyter:
-    image: nbgallery/jupyter-alpine
+    image: nbgallery/base-notebook
     ports:
-      - "80:80"
+      - "80:8888"
     environment:
       NBGALLERY_URL: http://localhost:3000
       NBGALLERY_CONFIG_TOKEN: ${NBGALLERY_CONFIG_TOKEN}
```

Comment:
base-notebook does not support NBGALLERY_CONFIG_TOKEN or NBGALLERY_CONFIG_PASSWORD.  @jeffyjefflabs said he might ad support for it in the future (https://github.com/nbgallery/docker-images/issues/8#issuecomment-531998137) but it is not a current issue, so should probably just remove these from the docker-compose if we are using base-notebook.

Reply:
Agreed